### DISCUSSION TITLE
按照要求改完配置环境后执行安装，配置环境的代码会自动复原

### DISCUSSION THREAD

**Discussion by J14N4N** [LmeSzinc/AzurLaneAutoScript#1361] [Q&A]
按照要求改完配置环境后执行安装，配置环境的代码会自动复原。请问怎么解决？也尝试在执行安装程序的改两处代码，但是执行完安装后代码依旧会自动恢复。也没弹出正常的使用界面。

**Comment by LmeSzinc** [comment]
改完了发截图来看看改对了没有，格式不对的话是会被还原成默认的
